packages/world/src/worldResourceTypes.sol

```diff
@@ -14,17 +14,9 @@ import { RESOURCE_TABLE, RESOURCE_OFFCHAIN_TABLE } from "@latticexyz/store/src/s
 // A namespace can include tables and systems.
 bytes2 constant RESOURCE_NAMESPACE = "ns";
 
-// Resource that identifies a module, an on-chain script that
-// can be executed on the World. Modules are used to install tables,
-// systems, hooks, and new entry point to a World in order to extend
-// its capabilities.
-bytes2 constant RESOURCE_MODULE = "md";
```

Comment:
@alvrs is it true that we don't use this anywhere?

Reply:
i don't think we do! modules are not exactly resources that are installed into the world either like other kinds of resources (namespaces, tables, systems)